pkg/defaults/defaults.go

```diff
@@ -150,6 +150,10 @@ const (
 	// This is used in DaemonConfig.Populate
 	ToFQDNsMinTTL = 0
 
+	// ToFQDNsActiveConnectionsTTL is the default value for how long that connections
+	// should be idle for after the DNS TTL expires.
+	ToFQDNsActiveConnectionsTTL = 120 * time.Second
```

Comment:
One thing that I am concerned about is that this default is set as a constant, without any reference to the `DNSGCJobInterval` value. But the CT GC and the Name Manager GC work together as per the comment in `pkg/fqdn/name_manager_gc.go`:

```
// activeConnections holds DNSName -> single IP entries that have been
// marked active by the CT GC. Since we expire in this controller, we
// give these entries 2 cycles of TTL to allow for timing mismatches
// with the CT GC.
```

So:

1) is it safe to set `ActiveConnectionsTTL` to a value lower than `2 * DNSGCJobInterval`? My understanding is that this can cause unexpected evictions of <name, IP> entries which should be kept alive because of an existing established connection.
2) what if in the future we change the `DNSGCJobInterval` value? I think it will be very easy to forget to adjust this related value too.

I think we can solve this keeping the default as `2 * DNSGCJobInterval`, thus moving the `DNSGCJobInterval` constant declaration here.